admin-offline/src/main/java/ch/ge/ve/offlineadmin/controller/BallotDecryptionController.java

```diff
@@ -97,9 +95,9 @@
     private StreamHasher streamHasher;
     private OutputFilesPattern outputFilesPattern;
     private PropertyConfigurationService propertyConfigurationService;
-    private PeriodFormatter periodFormatter;
     private PasswordDialogController passwordDialogController;
     private BallotCipherServiceFactory ballotCipherServiceFactory;
+    private ZoneId ChZoneId = ZoneId.of("Europe/Zurich");
```

Comment:
The variable name is not consistent with the naming of the other variables.
Would you consider changing it to `chZoneId` ?